mbed_greentea/mbed_target_info.py

```diff
@@ -92,6 +92,20 @@
                 "reset_method": "default",
                 "program_cycle_s": 4
             }
+        },
+    "ARCH_BLE" : {
+        "yotta_targets": [
+                {
+                    "yotta_target": "tinyble-gcc",
```

Comment:
target is called as the platform (tinyble) ? And I believe ARCH_BLE is the name coming from mbed-ls?